<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QStackedWidget" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>北京国广电科技有限公司</string>
  </property>
  <widget class="QWidget" name="page">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="title">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#title
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.0113636, y2:1, stop:0 rgba(67, 166, 245, 255), stop:0.994318 rgba(0, 105, 204, 255));
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 22pt &quot;黑体&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>音频链路监测服务端</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true"/>
       </item>
       <item>
        <widget class="QLCDNumber" name="nowTime">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="digitCount">
          <number>19</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="control">
        <property name="maximumSize">
         <size>
          <width>105</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 100, 255);</string>
        </property>
        <item>
         <property name="text">
          <string>音频录制</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>设备参数</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>录音参数</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>运行图设置</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>录音回放</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>录音备份</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>录音上传</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>运行图参数</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>运行图查看</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>帮助</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="page_ly_state">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>915</width>
               <height>697</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_set"/>
        <widget class="QWidget" name="page_saveSet"/>
        <widget class="QWidget" name="page_yxt"/>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
